### DISCUSSION TITLE
Supabase Pricing Question - Transfer Limits

### DISCUSSION THREAD

**Discussion by ryanjewsburyMMU** [supabase/supabase#6268] [Questions] [answered]
So I'm new to supabase & creating my first project for a family business. 

I was wondering how the transfer limits work? I'm a bit confused on the actual functions - we get 'unlimited api requests', but under that it says transfer limits of 2GB. So my questions are: 

- If we make n api request does this effect your transfer limit? 

Because on the paid version of £25 a month, the transfer limit only goes up by 6GB. 

Or does it mean, at any one time, we could only make a request that would bring x amount of data from the db. 

If anyone could explain this in a bit more detail, i would appreciate it! Cheers :)

**Comment by GaryAustin1** [comment, marked as answer]
In free plan 2GB is download data in one(!) or thousands of calls per month.
I think you are confusing 8GB in the paid plan... that is Database size (from 500MB to 8GB).
Download transfer included in the paid plan is 50GB for database (per month).

**Reply by ryanjewsburyMMU** [reply]
Yes your right, my bad - so it goes from 2GB to 50GB. 
So it says you have unlimited api calls, but realistically, you only have x amount of api calls up until you've made calls of 2gb's worth of data?

**Reply by GaryAustin1** [reply]
Yes.  Upload does not count if I remember.
Basically they are saying (unlike Firebase) they don't bill per api call.

**Reply by ryanjewsburyMMU** [reply]
Thanks for your help! So is there any way to monitor this?

**Reply by GaryAustin1** [reply]
There is a report page you can set with what you want:
![image](https://user-images.githubusercontent.com/54564956/161393056-8dfbe158-42e1-4786-9829-d80301d68c4d.png)

**Reply by ryanjewsburyMMU** [reply]
Thats brilliant. Thanks so much for your help.

**Reply by dgbkn** [reply]
Is it mean that after 2 gb of egress api will be stopped in a month???

**Reply by GaryAustin1** [reply]
@dgbkn Basically yes.  It is a soft limit, but if you are exceeding that regularly you need to move to the next step.
